markdown/chapters/meandeck/interaction.md

```diff
@@ -522,10 +550,66 @@ cantrips. You can also build a pile around it *if* you can predict when the
+resolving or ever triggering in the first place. For any permanent based one you
+will need to remove it to be able to win from the Oracle trigger. For the
+instant speed ones, you will need some sort of interaction, either counter magic
+or discard (however in the case of {{Trickbind}} you have to be proactive thanks
+to the split second ability) so something like Duress is a must. You can also
+consider using a second Oracle in your pile if you only suspect a single use
+example of these effects.
+
+### Thoughtseize / Thoughtknot Seer / Cabal Therapy
+
+These effects tend to only be relevant for PTT piles. Normally they can be
+safely ignored however they may cause delay to your pile winning or, in the
+worst case scenario, could take your Oracle out of your hand leaving you unable
+to win. Just be mindful of when these cards could come up if building a PTT
+pile.
+
+### Ragavan, Nimble Pilferer
+
+Another PTT only example. You may know about a {{Ragavan, Nimble Pilferer}} on
+board or suspect your opponent might be able to *dash* one from their hand. When
+considering Ragavan you have to work out what cards in the pile they will be
+exiling with it.
+
+The safest method to deal with it is place a cycle effect in any slot they are
+likely to exile. This way if they do exile it, they cannot obtain any value from
+it and if they don't exile it you have still have the draw effect available to
+you.
+
+Consider the following example from the basics chapter.
+
+> {B}{B}{B} + {U}{U} + PTT
+
+<row variant="pile">{{!IU}} {{!LP}} {{!LP}} {{!SW}} {{!TO}}</row>
+
+You can make a minor adjustment without losing functionality of the pile by
+placing the SW as the first card.
+
+> {B}{B}{B} + {U}{U} + PTT
+
+<row variant="pile">{{!SW}} {{!IU}} {{!LP}} {{!LP}} {{!TO}}</row>
+
+In this way, even if you are attacked and hit by a Ragavan, you can still draw
+your entire pile on the following turn and cast Oracle.
+
 ## Final Thoughts
 
 There are a number of things that you may face and the ability to deal with it
 comes from research and practice. Remember to always consider the matchup in
-playing around Wasteland, opposing Thoughtseize effects and the like prior to
-the combo to give yourself the best chance to enact the combo. Consider whether
-speed or resilience is the right path given your opponent's deck.
+playing around Wasteland, opposing Thoughtseize effects, Daze and the like prior
+to the combo to give yourself the best chance to enact the combo. Consider
+whether speed or resilience is the right path given your opponent's deck.
+Because of the size of the Legacy card pool and the variance between decklists,
+even within proven archetypes, you can often face strange or unexpected corner
+cases. Just try your best to make note of what you are aware, play around what
+you can but be mindful that sometimes you cannot play around everything and need
+to accept if they have the one corner case card then you will not be able to
```

Comment:
I think it reads better if the sentence ends right after "the one corner case card"

Reply:
Changed